pypeit/images/pypeitimage.py

```diff
@@ -151,6 +165,7 @@ def _bundle(self):
 
         # Rest of the datamodel
         for key in self.keys():
+            # TODO: Why are these skipped?
```

Comment:
I forget..

Reply:
Given what is and is not output to the master files, I'm a little worried that reusing existing masters now loses some of the necessary variance arrays.  Will try to figure this out...

Reply:
The result of this is that I added the inverse variance arrays to the bias and dark frame output files so that they can be reused.